src/app.js

```diff
@@ -4,6 +4,7 @@
 // @flow
 
 import Koa from 'koa';
+import helmet from 'koa-helmet';
```

Comment:
documentation for koa-helmet can be found at https://github.com/venables/koa-helmet